.gitmodules

```diff
@@ -1,3 +1,5 @@
 [submodule "lib/impUnit"]
 	path = lib/impUnit
 	url = https://github.com/electricimp/impUnit.git
+	branch = develop
```

Comment:
Instead of "develop" is there a way to point at a specific commit?

Reply:
I am no expert in this, but going off of the StackOverflow answer that told me how to set it to a branch - the answer is no. You can only set it to a branch.

That being said, when you do the submodule update thing it locks in the commit from the branch you're pointing to. So even if new changes are made to that `develop` branch, it won't affect us until someone comes in and manually does a submodule update and then commits the change.

Reply:
OK, I think that was a total lie. It _appeared_ to lock it in ... but that was only in my current checkout. If I throw that away and check it out fresh it looks like it's just going to pull the latest from develop.

Given the very, very low frequency of changes to that develop branch, I think it's worth making the change though for now.

@mm-gmbd if you want to push them to put out a new release, then we could switch this back over to master at that point – but for right now I think it would be good to get everyone using the version that has the changes/fixes without having to wait for ElectricImp.